packages/shared/lib/core/profile/actions/active-profile/getNetworkHrp.ts

```diff
@@ -0,0 +1,7 @@
+import { activeProfile } from '@core/profile/stores'
+import { get } from 'svelte/store'
+
+export function getNetworkHrp(): string {
```

Comment:
It feels strange that this function isn't in the `network` module - it may be better to put it there and have it take a `IProfile` as an argument and return the HRP.

Reply:
Also this is more of a `util` function rather than an `action`, especially if we end up changing it to take any profile object as an argument.

Reply:
utils should not be getting anything from a store. So its an action.

Reply:
This can be refactored in l2 prs for networking, but for now its stored on the active profile and we need a quick way to get it, without having to pass the active profile in everywhere